README.md

```diff
@@ -1,30 +1,42 @@
-# AHM dry run scripts
+# TL;DR
 
-Scripts for dry-running the Asset Hub migration and verifying the post-migration state.
-
-Using [Chopsticks](https://github.com/AcalaNetwork/chopsticks) and [PAPI](papi.how).
-
-## Getting started
+To run AHM for Paseo using Zombie-Bite:
 ```
-git clone --recursive git@github.com:paritytech/ahm-dryrun.git && cd ahm-dryrun
+git clone --recursive git@github.com:paritytech/ahm-dryrun.git && \
+cd ahm-dryrun && \
+just init && \
+just setup && \
+just ahm paseo || echo "Setup failed"
 ```
 
-Or, if you've already cloned and didn't use the recursive flag, you can run:
```

Comment:
So if one has already cloned the repo, the above still stands? (Excluding the `git clone`, of course)